static/lang/re-en.json

```diff
@@ -3661,6 +3661,7 @@
                 "Color": "Select a Color.",
                 "ConsciousMind": "Select a conscious mind.",
                 "CreatureSize": "Select a creature size.",
+                "CreatureTrait": "Select the summoned or manifested creature trait.",
```

Comment:
That's really long for a roll option label, Something like "Ostentatious Arrival Creature Trait" would work instead.

Reply:
Alright, changed - check when you get a moment.